src/mscorlib/src/System/Globalization/DateTimeFormatInfo.cs

```diff
@@ -2881,7 +2881,7 @@ private void AddMonthNames(TokenHashValue[] temp, String monthPostfix)
                                 // - if the app is targetting 4.5.1 or above OR the compat flag is set, use the correct behavior by default.
                                 // - if the app is targetting 4.5 or below AND the compat switch is set, use the correct behavior
                                 // - if the app is targetting 4.5 or below AND the compat switch is NOT set, use the incorrect behavior
-                                if (preferExistingTokens || BinaryCompatibility.TargetsAtLeast_Desktop_V4_5_1)
+                                if (preferExistingTokens)
```

Comment:
shouldn't this be the other way around? preferExistingTokens is always false on coreclr so you'd use the incorrect behavior in the else as per the comment above.